snowflake_data_profiler/profiling/profiler.py

```diff
@@ -46,14 +48,22 @@ def connect_to_snowflake(sfUser, sfPswd, sfURL, sfDatabase, sfSchema, sfTable, s
         schema=sfSchema,
         role=sfRole,
     )
+    return con
+
+
+# creates cursor object and returns a pandas dataframe
+def create_cursor(con, sfDatabase, sfSchema, sfTable, sfWarehouse=None):
     cur = con.cursor()
     if sfWarehouse:
         cur.execute(f'use warehouse {sfWarehouse};')
 
     cur.execute(f'select * from {sfDatabase}.{sfSchema}.{sfTable} limit 10000;')
     df = cur.fetch_pandas_all()
+
     return df
 
+
+# profiles pandas dataframe
```

Comment:
Let's start using docstrings for functions:

https://www.geeksforgeeks.org/python-docstrings/

We can discuss this if it's confusing at first